components/error_page/error_message.tsx

```diff
@@ -1,16 +1,21 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
 import {ErrorPageTypes} from 'utils/constants';
 import {t} from 'utils/i18n';
 
-import ErrorLink from './error_link.jsx';
+import ErrorLink from './error_link';
 
-export default function ErrorMessage({type, message, service}) {
+type Props = {
+    type: string | null;
```

Comment:
better rewrite it as `type?: string`

Reply:
also, message and service should be marked optional

Reply:
The reason I wrote it like that is because of this: https://circleci.com/gh/mattermost/mattermost-webapp/33402?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link type check failing. Shall I revert it back to `type: string | null;`?